identityClient/api.go

```diff
@@ -1219,11 +1221,15 @@ type DeletePAMJiraPluginRequest = PAMJiraPluginByIDRequest
 // DescribePAMJiraPluginRequest wraps necessary data for describing an existing Jira integration
 type DescribePAMJiraPluginRequest = PAMJiraPluginByIDRequest
 
+// TestPAMJiraPluginRequest wraps necessary data for testing an existing Jira integration's credentials
+type TestPAMJiraPluginRequest = PAMJiraPluginByIDRequest
```

Comment:
test is a confusing prefix for this struct and method, prefer something like `Ping` or `HealthCheck`

Reply:
updated everything to be "ping"